fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/node/CosExpNode.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.database.utils.query.node;
```

Comment:
small suggestion - moving the math functions to a sub package

Reply:
#2321